src/main/proto/org/hyades/repometaanalysis/v1/repo_meta_analysis.proto

```diff
@@ -11,6 +11,15 @@ option java_package = "org.hyades.proto.repometaanalysis.v1";
 message AnalysisCommand {
   // The component that shall be analyzed.
   Component component = 1;
+  FetchMeta fetch_meta = 2;
+
+}
+
+enum FetchMeta{
+  FETCH_META_UNSPECIFIED = 0;
```

Comment:
Do we need FETCH_META_UNSPECIFIED? as it should be either one or both but never unspecified. If it is unspecified, the event wouldn't be called at all.

Reply:
It is part of the proto enum standard. Would not satisfy the enum standard without the unspecified enum field